space-cli/cmd/modules/accounts/set.go

```diff
@@ -0,0 +1,59 @@
+package accounts
+
+import (
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/spaceuptech/space-cli/cmd/model"
+	"github.com/spaceuptech/space-cli/cmd/utils"
+	"github.com/spaceuptech/space-cli/cmd/utils/input"
+)
+
+func setAccount(prefix string) error {
+	credential, err := utils.GetCredentials()
+	if err != nil {
+		return err
+	}
+
+	accountIDs := []string{}
+	for _, v := range credential.Accounts {
+		accountIDs = append(accountIDs, v.ID)
+	}
+
+	prefix = strings.ToLower(prefix)
+	filteredAccountIDs, doesAccountExists := filterAccounts(credential.Accounts, prefix)
+
+	if doesAccountExists {
```

Comment:
You don't need this variable. FilterAccounts should returb the appropriate array

Reply:
It does return the approriate array but I need this variable to distinguish between accounts matching or not. I can change it to check the length of filteredOptions maybe so no new variable is needed or shift the no account matching to the filter function. What do I do?